lib/alchemy/upgrader/four_point_zero.rb

```diff
@@ -1,3 +1,104 @@
+require 'thor'
+
+class Alchemy::Upgrader::FourPointZeroTasks < Thor
+  include Thor::Actions
+
+  no_tasks do
+
+    def convert_available_contents_to_nestable_elements
+      old_config_file = Rails.root.join('config', 'alchemy', 'elements.yml')
+      print "Reading config/alchemy/elements.yml ... "
+      elements_config = YAML.load_file(old_config_file)
+      puts "done."
+
```

Comment:
We should exit here, if `elements_with_available_contents` is empty, because there is nothing to do. 

So collecting `elements_with_available_contents` should be done at first.